stock_picking_invoice_link/views/stock_view.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- stock.picking -->
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="name">stock_picking_invoice_link.stock.picking.form</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <field name="state" position="before">
+                    <button name="action_view_invoice" type="object"
+                        string="View Invoice" class="oe_highlight"
+                        attrs="{'invisible': [('invoice_ids', '=', [])]}"
+                        groups="base.group_user"/>
+            </field>
+            <field name="picking_type_id" position="after">
+                <field name="invoice_ids" string="Invoice"
```

Comment:
Wouldn't a tree view be mote practical?

Reply:
I think not. View adds two items to show invoice information in picks:
- a button to open invoice/s directly
- a field to quickly see if picking is invoiced.
I think is enough information:
![seleccio_329](https://user-images.githubusercontent.com/4386443/30322453-ce08b35e-97b9-11e7-8000-51ca62cab198.png)

Reply:
Oh, I didn't notice the button, sorry